drivers/npmx/npmx_shell.c

```diff
@@ -106,2595 +146,1919 @@ static bool check_error_code(const struct shell *shell, npmx_error_t err_code)
 	return false;
 }
 
-static bool check_pin_configuration_correctness(const struct shell *shell, int8_t gpio_idx)
+static const char *unit_str_get(unit_type_t unit_type)
 {
-	int8_t pmic_int_pin = (int8_t)npmx_driver_int_pin_get(pmic_dev);
-	int8_t pmic_pof_pin = (int8_t)npmx_driver_pof_pin_get(pmic_dev);
-
-	if ((pmic_int_pin != -1) && (pmic_int_pin == gpio_idx)) {
-		shell_error(shell, "Error: GPIO used as interrupt.");
-		return false;
+	switch (unit_type) {
+	case UNIT_TYPE_MILLIAMPERE:
+		return " mA";
+	case UNIT_TYPE_MILLIVOLT:
+		return " mV";
+	case UNIT_TYPE_CELSIUS:
+		return "*C";
+	case UNIT_TYPE_OHM:
+		return " ohm";
```

Comment:
According to @annakielar correct form is "ohms"

Reply:
Changed.